spec/phut/parser_spec.rb

```diff
@@ -2,55 +2,63 @@
 
 describe Phut::Parser do
   describe '#parse' do
-    subject { Phut::Parser.new.parse configuration }
+    Given(:config) { Phut::Parser.new.parse string }
 
     context "with 'vswitch { dpid '0xabc' }'" do
-      let(:configuration) { "vswitch { dpid '0xabc' }" }
+      When(:string) { "vswitch { dpid '0xabc' }" }
 
-      it { expect { subject }.not_to raise_error }
+      describe '#vswitch' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.